docs/framework/guides/tutorials/implementing-a-block-widget.md

```diff
@@ -969,10 +969,10 @@ ClassicEditor
 		toolbar: [ 'heading', 'bold', 'italic', 'numberedList', 'bulletedList', 'simpleBox' ]
 	} )
 	.then( editor => {
-		// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.